data/annual.csv

```diff
@@ -1,275 +1,320 @@
-GISTEMP,1956,-0.2
-GCAG,1955,-0.1354
-GISTEMP,1955,-0.15
-GCAG,1954,-0.1165
-GISTEMP,1954,-0.13
-GCAG,1953,0.0952
-GISTEMP,1953,0.08
-GCAG,1952,0.0248
-GISTEMP,1952,0.01
-GCAG,1951,-0.0132
-GISTEMP,1951,-0.07
-GCAG,1950,-0.1616
-GISTEMP,1950,-0.18
-GCAG,1949,-0.0568
-GISTEMP,1949,-0.09
-GCAG,1948,-0.0487
-GISTEMP,1948,-0.09
-GCAG,1947,-0.0477
-GISTEMP,1947,-0.05
-GCAG,1946,-0.0040
-GISTEMP,1946,-0.04
-GCAG,1945,0.1710
-GISTEMP,1945,0.12
-GCAG,1944,0.2928
-GISTEMP,1944,0.25
-GCAG,1943,0.1570
-GISTEMP,1943,0.13
-GCAG,1942,0.1538
-GISTEMP,1942,0.09
-GCAG,1941,0.1960
-GISTEMP,1941,0.12
-GCAG,1940,0.0947
-GISTEMP,1940,0.08
-GCAG,1939,-0.0139
-GISTEMP,1939,-0.03
-GCAG,1938,-0.0288
-GISTEMP,1938,-0.03
-GCAG,1937,-0.0157
-GISTEMP,1937,-0.03
-GCAG,1936,-0.1134
-GISTEMP,1936,-0.15
-GCAG,1935,-0.1392
-GISTEMP,1935,-0.2
-GCAG,1934,-0.1015
-GISTEMP,1934,-0.14
-GCAG,1933,-0.2439
-GISTEMP,1933,-0.29
-GCAG,1932,-0.1168
-GISTEMP,1932,-0.17
-GCAG,1931,-0.0686
-GISTEMP,1931,-0.09
-GCAG,1930,-0.1003
-GISTEMP,1930,-0.15
-GCAG,1929,-0.2985
-GISTEMP,1929,-0.36
-GCAG,1928,-0.1774
-GISTEMP,1928,-0.21
-GCAG,1927,-0.1546
-GISTEMP,1927,-0.21
-GCAG,1926,-0.0667
-GISTEMP,1926,-0.1
-GCAG,1925,-0.1481
-GISTEMP,1925,-0.21
-GCAG,1924,-0.2486
-GISTEMP,1924,-0.28
-GCAG,1923,-0.2156
-GISTEMP,1923,-0.24
-GCAG,1922,-0.2304
-GISTEMP,1922,-0.28
-GCAG,1921,-0.1485
-GISTEMP,1921,-0.21
-GCAG,1920,-0.2105
-GISTEMP,1920,-0.27
-GCAG,1919,-0.2055
-GISTEMP,1919,-0.22
-GCAG,1918,-0.2084
-GISTEMP,1918,-0.26
-GCAG,1917,-0.3146
-GISTEMP,1917,-0.4
-GCAG,1916,-0.2930
-GISTEMP,1916,-0.34
-GCAG,1915,-0.0693
-GISTEMP,1915,-0.11
-GCAG,1914,-0.1395
-GISTEMP,1914,-0.16
-GCAG,1913,-0.3162
-GISTEMP,1913,-0.34
-GCAG,1912,-0.3288
-GISTEMP,1912,-0.35
-GCAG,1911,-0.4332
-GISTEMP,1911,-0.44
-GCAG,1910,-0.3789
-GISTEMP,1910,-0.42
-GCAG,1909,-0.4261
-GISTEMP,1909,-0.47
-GCAG,1908,-0.4396
-GISTEMP,1908,-0.43
-GCAG,1907,-0.3706
-GISTEMP,1907,-0.4
-GCAG,1906,-0.2174
-GISTEMP,1906,-0.23
-GCAG,1905,-0.2931
-GISTEMP,1905,-0.28
-GCAG,1904,-0.4194
-GISTEMP,1904,-0.44
-GCAG,1903,-0.3369
-GISTEMP,1903,-0.35
-GCAG,1902,-0.2463
-GISTEMP,1902,-0.27
-GCAG,1901,-0.1417
-GISTEMP,1901,-0.15
-GCAG,1900,-0.0679
-GISTEMP,1900,-0.09
-GCAG,1899,-0.1173
-GISTEMP,1899,-0.16
-GCAG,1898,-0.2546
-GISTEMP,1898,-0.28
-GCAG,1897,-0.1224
-GISTEMP,1897,-0.11
-GCAG,1896,-0.0974
-GISTEMP,1896,-0.15
-GCAG,1895,-0.2290
-GISTEMP,1895,-0.21
-GCAG,1894,-0.2808
-GISTEMP,1894,-0.31
-GCAG,1893,-0.3212
-GISTEMP,1893,-0.3
-GCAG,1892,-0.3062
-GISTEMP,1892,-0.27
-GCAG,1891,-0.2532
-GISTEMP,1891,-0.24
-GCAG,1890,-0.3220
-GISTEMP,1890,-0.37
-GCAG,1889,-0.0982
-GISTEMP,1889,-0.12
-GCAG,1888,-0.1471
-GISTEMP,1888,-0.2
-GCAG,1887,-0.2489
-GISTEMP,1887,-0.33
-GCAG,1886,-0.2003
-GISTEMP,1886,-0.31
-GCAG,1885,-0.2125
-GISTEMP,1885,-0.32
-GCAG,1884,-0.2009
-GISTEMP,1884,-0.28
-GCAG,1883,-0.1424
-GISTEMP,1883,-0.21
-GCAG,1882,-0.0648
-GISTEMP,1882,-0.1
-GCAG,1881,-0.0628
-GISTEMP,1881,-0.12
-GCAG,1880,-0.1148
-GISTEMP,1880,-0.2
+Source,Year,Mean
+gcag,1850,-0.4177

```

Comment:
@gradedSystem can we keep source name in upper case as in the old data file - `GCAG`? Same applied to the monthly data file.

Reply:
Yes I forgot to write it in UPPERCASE